webServer.js

```diff
@@ -255,6 +257,15 @@ function start() {
     });
   }
 
+  function libraryMetaInfo(library) {
+    if (library.repository === undefined) {
```

Comment:
I use `/github.com/.test(library.repository.url)` to check if it's Github url. :+1: